### PR TITLE
Fixes #7941: Display sync status correctly on repository info.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Anyone available to review this?

**Comment by cfouant** [comment]
Can you rebase and I will review this p-r?

**Comment by ehelms** [comment]
Rebased @cfouant

**Comment by cfouant** [comment]
@ehelms - It doesn't seem to work if the repo has never been synced, but works nicely for anything with sync history. Can you put in a handling of this scenario?

File: lib/hammer_cli_katello/repository.rb [hunk 1]
```diff
@@ -73,7 +73,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
       def extend_data(data)
         data["_redhat_repo"] = data["product_type"] == "redhat" ? _("yes") : _("no")
         data["_publish_via_http"] = data["unprotected"] ? _("yes") : _("no")
-        data["_sync_state"] = get_sync_status(data["sync_state"])
+        data["_sync_state"] = get_sync_status(data["last_sync"]["result"])
```
Comment (cfouant) [inline lib/hammer_cli_katello/repository.rb]: I think you want `data["_sync_state"] = get_sync_status(data["last_sync"]["state"])` here

**Comment by daviddavis** [comment]
@ehelms ping?

**Comment by ehelms** [comment]
Updated to also properly populate the last sync date.

File: lib/hammer_cli_katello/repository.rb [hunk 2]
```diff
@@ -73,11 +73,20 @@ class InfoCommand < HammerCLIKatello::InfoCommand
...
+          data['last_sync'] = data['last_sync_words']
+        else
+          data['_sync_state'] = _("Not Synced")
+        end
+
         if data["content_type"] == "yum" && data["gpg_key"]
           data["gpg_key_name"] = data["gpg_key"]["name"]
         end
 
+        require 'pp';pp data
```
Comment (daviddavis) [inline lib/hammer_cli_katello/repository.rb]: Remove me.

**Comment by cfouant** [comment]
ACK

**Comment by ehelms** [comment]
Thanks for the reviews @cfouant and @daviddavis

**Pull request merged**